Restore failure on creation of wpflib template

* .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`)

```
.NET Core SDK (reflecting any global.json):
 Version:   3.0.100-preview8-013640
 Commit:    1d6c76dc2b

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.0.100-preview8-013640\

Host (useful for support):
  Version: 3.0.0-preview8-28380-08
  Commit:  c855ac7187

.NET Core SDKs installed:
  2.2.401 [C:\Program Files\dotnet\sdk]
  3.0.100-preview8-013437 [C:\Program Files\dotnet\sdk]
  3.0.100-preview8-013640 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.0-preview8.19374.1 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.0-preview8.19381.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0-preview8-28373-17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0-preview8-28380-08 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.0.0-preview8-28373-17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.0.0-preview8-28380-08 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

* Windows version: (`winver`)

Windows 10, 1903 RTM

* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes/No

N/A
 
 **Problem description:**

Get an error when running the following to create a new WPF library VB project:

 `dotnet new  wpflib --language VB -o repro`

 **Actual behavior:** <!-- callstack for crashes / exceptions -->

Restore tries to run on Company.ClassLibrary1.csproj, instead of the created repro.vbproj.
The project doens't exist so it fails.
 
```
The template "WPF Class library" was created successfully.

Processing post-creation actions...
Running 'dotnet restore' on repro\Company.ClassLibrary1.csproj...
MSBUILD : error MSB1009: Project file does not exist.
Switch: repro\Company.ClassLibrary1.csproj

Restore failed.
Post action failed.
Description: Restore NuGet packages required by this project.
Manual instructions: Run 'dotnet restore'
```

 **Expected behavior:**

No errors from post-creation restore
 
 **Minimal repro:**
 
 `dotnet new  wpflib --language VB -o repro`

@jmarolf @vatsan-madhavan 

Found while trying all the VB templates for something else.

--- a/packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-VisualBasic/.template.config/template.json
+++ b/packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfClassLibrary-VisualBasic/.template.config/template.json
@@ -55,7 +55,7 @@
     }
   },
   "primaryOutputs": [
-    { "path": "Company.ClassLibrary1.csproj" },
+    { "path": "Company.ClassLibrary1.vbproj" },
     {
       "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
       "path": "Class1.vb"
@@ -74,7 +74,7 @@
     },
     {
       "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
-      "description": "Opens Class1.cs in the editor",
+      "description": "Opens Class1.vb in the editor",
       "manualInstructions": [ ],
       "actionId": "84C0DA21-51C8-4541-9940-6CA19AF04EE6",
       "args": {

